### PR TITLE
Camera

### PR DESCRIPTION
**Pull request by Muqsit**
It's up to you to merge this :)
screenshot_2016-02-21-02-23-33

### CODE REVIEW

**Comment by gaoyichuan** [comment]
I don't think we should add this thing which isn't existed in MCPE now.. @iTXTech/developers

**Comment by BlackShadow1** [comment]
@gaoyichuan http://minecraft.gamepedia.com/Camera
Look at history.

**Comment by Muqsit** [comment]
It has no function as of now. So, it's upto you to merge or close :P

File: src/pocketmine/block/Camera.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		$up = $this->getSide(1);
+		if($down->getId() === self::GRASS or $down->getId() === self::DIRT){
+			$this->getLevel()->setBlock($block, $this, true);
+			$this->getLevel()->setBlock($up, Block::get($this->id, $this->meta ^ 0x08), true);
+			return true;
+		}
+		return false;
+	}
+
+	public function onBreak(Item $item) : array{
```
Comment (xpyctum) [inline src/pocketmine/block/Camera.php]: array?

**Comment by gaoyichuan** [comment]
@BlackShadow1 Oh, I didn't know it is added back.. sorry for my former words

**Comment by BlackShadow1** [comment]
@gaoyichuan No problem, next time check if it's on the game before writing. :P

**Comment by gaoyichuan** [comment]
Ok, xD

**Comment by Muqsit** [comment]
Thanks for commenting on my PR...makes me feel special :3

**Comment by gaoyichuan** [comment]
@Muqsit At the time when I was not a collaborator I felt exactly the same when my PR was merged.xD

**Comment by Muqsit** [comment]
@gaoyichuan xD

**Comment by qq1281713718** [comment]
Good!

**Comment by Muqsit** [comment]
Camera was removed in PE 0.16. It's only available in Minecraft Education Edition now.

**Pull request merged**